src/Gambot.Module.Factoid/Factoid.cs

```diff
@@ -5,7 +5,7 @@ public class Factoid
         public string Trigger { get; set; }
         public string Verb { get; set; }
         public string Response { get; set; }
-
-        public override string ToString() => $"{Trigger} <{Verb}> {Response}";
+        public decimal PercentChanceToProc { get; set; }

```

Comment:
Please call this something normal you f'ckin nerd

Reply:
`ChanceToProc`?

Reply:
i wanted to make sure `Percent` was in the name too to make it very obvious that its stored as `30` rather than `0.3` for 30%

Reply:
I was referring to the "proc" part, really.  This ain't WoW.  Also the rest of the factoid code uses "trigger" to refer to the initiating condition.

I'm not sure how useful it is to make this distinction obvious here, since it's not a named configuration value - most people will never see this name, and for devs a comment could do just fine (worth noting is that percentages are stored out of 100 in the rest of the Gambot codebase).

Reply:
ah, fair enough -- `ChanceToTrigger` seems like itd fit well